### ISSUE TITLE
BUG: Error in @context  cite-as does not resolve

### ISSUE THREAD

**Issue by ptsefton** [ResearchObject/ro-crate#239]
**Describe the bug**
 The default context has a definition of cite-as which does not resolve. https://www.w3.org/ns/iana/link-relations/relation#cite-as

(I want to use this property to put a text citation on a crate so it can say "Cite this crate as ...")

**URL**
https://www.researchobject.org/ro-crate/1.1/context.jsonld

**Comment by ptsefton** [comment]
@stain what do you think, can we use preferredCitation from the Library of Congress?

**Comment by stain** [comment]
I think our namespace for link relations is wrong or outdated.

https://www.iana.org/assignments/link-relations/link-relations.xhtml is the official IANA registry, which defines "cite-as" as of [RFC8574](https://doi.org/10.17487/rfc8574) (April 2019) but W3C's (another Standard body) https://www.w3.org/ns/iana/link-relations/relation was last updated 2 years before that in 2017.

```
(base) stain@xena11:~$ curl -I 'https://www.w3.org/ns/iana/link-relations/relation' | grep -i last-modified
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0 32852    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
last-modified: Tue, 24 Oct 2017 21:31:10 GMT
```

Pinging @hvdsomp on what may be the right namespace. 

@dret also have https://webconcepts.info/concepts/link-relation/cite-as which I think is an unofficial concept registry covering https://webconcepts.info/concepts/link-relation/ -- it has a nicer landing page than the outdated W3 Turtle.

**Comment by stain** [comment]
I would prefer to stay with `cite-as` so we are aligned with [FAIR Signposting](https://signposting.org/FAIR/) which we are using with RO-Crate . 

We know http://schema.org/citation is a bit weak (it could be a citation for a paper about the thing, or a paper that the thing cites, rather than citable identifier for the RO-Crate as a digital object) and http://schema.org/identifier not specific enough unless you inspect the string for `https://doi.org/` (which is too narrow on identifier providers)

**Comment by stain** [comment]
http://www.iana.org/assignments/relation/cite-as redirects correctly (but not to right row), the namespace `http://www.iana.org/assignments/relation/` (note no **s**) was [suggested by RFC5988](https://www.rfc-editor.org/rfc/rfc5988.html#appendix-B) and updated in [RFC8288 A2](https://www.rfc-editor.org/rfc/rfc8288.html#appendix-A.2) suggested for use by Atom format, -- so perhaps we should use the same here.

**Comment by hvdsomp** [comment]
The core problem here is that there is no official way to express well-defined IANA link relation types (all are strings) as URIs. This issue has come up time and again and, relatively recently, I have lamented about it in the context of work for [RFC9264](https://www.rfc-editor.org/rfc/rfc9264.txt) Link Set, see https://github.com/ietf-wg-httpapi/linkset/issues/45#issuecomment-925865896. In 2016, it was discussed at length in https://github.com/mnot/I-D/issues/140. The pragmatic approach is the one that @stain proposes, i.e. use `http://www.iana.org/assignments/relation/` as the URL to which an IANA link relation type (string) listed at [https://www.iana.org/assignments/link-relations/link-relations.xhtml](https://www.iana.org/assignments/link-relations/link-relations.xhtml) is appended. This is, BTW, also the approach used in [COAR Notify](https://notify.coar-repositories.org/) as can be seen from the [JSON-LD context file](https://notify.coar-repositories.org/schema/notify.json) used by the protocol. In COAR Notify, [FAIR Signposting](https://signposting.org/FAIR/) is also an important building block for interoperability; hence their use of `cite-as`, which explicitly has the semantics desired by @ptsefton as can be seen from [RFC8574](https://www.rfc-editor.org/rfc/rfc8574.txt).

**Comment by ptsefton** [comment]
Should we add this to the [section on the root dataset](https://github.com/ResearchObject/ro-crate/blob/master/docs/1.2-DRAFT/root-data-entity.md) in addition to setting a DOI in the identifier property?

Also do we need  a way to add a TEXT statement of how to cite - something like a datacite citation not a link?

**Comment by stain** [comment]
This should be addressed by #255, awaiting review.
